src/components/link.js

```diff
@@ -67,6 +67,8 @@ export default {
       ? classes[exactActiveClass]
       : isIncludedRoute(current, compareTarget)
 
+    const ariaCurrentValue = classes[exactActiveClass] ? 'page' : null
```

Comment:
I think it makes sense to allow customising the value of `page`. Can we add an optional prop `ariaCurrentValue` that defaults to `page`

Reply:
@posva that could make sense but the other options for `aria-current` aren't really related to links.

Reply:
`step` (and a few others) could, so allowing to customise it makes sense

Reply:
yeah I suppose. If you have a stepper component that uses vue-router and changes the url, that would make sense.

Reply:
is `ariaCurrentValue` fine as the prop name? Happy to add as long as it's uncontroversial 😄

Reply:
Seems fine to me. It's similar to `true-value` and `false-value` and it's very explicit. So unless anybody has a better suggestion we should go with it